source/hardware/controllers/reloop_beatmix.rst

```diff
@@ -0,0 +1,80 @@
+.. _reloop-beatmix:
+
+Reloop Beatmix
+==============
+
+This is a compact sized two decker controller with two stereo audio outputs and a mic input on board.
+It features two 14cm touch sensitive jogs with adjustable sensitivity, gain dials, filter/fx controls and mouseless library navigation dial.
+Most of controls have a second function accessible through corresponding Shift buttons.
+
+-  `Manufacturer’s product page <https://www.reloop.com/reloop-beatmix>`__
+-  `Forum thread <https://mixxx.discourse.group/t/reloop-beatmix-1st-gen-mapping-implemented/28536>`__
+
+.. versionadded:: 2.4
+
+Audio
+-----
+
+This controller is a class complaint USB audio device. :term:`ASIO` driver for Windows based setups provided by manufacturer.
+
+.. figure:: ../../_static/controllers/reloop_beatmix.png
```

Comment:
Do you mind putting bigger button/knob labels on top?
They're pretty hard to read in a regukar browser view.

Reply:
Ok, i will update the pic. It will require some work though. This current one is just a rip from user manual, i gonna need to make a new one myself.

Reply:
I mean it'll take some time to get it done :)

Reply:
I just thought about putting some slightly larger text bubbles on top of it with Gimp or alike, no need to create an entirely new image.
If it's too much effort, leave it as it is.